web/jobs/jobs_test.go

```diff
@@ -401,6 +401,27 @@ func TestGetAllJobs(t *testing.T) {
 		assert.Equal(t, "10s", v.Data[index].Attributes.Arguments)
 		assert.Equal(t, "print", v.Data[index].Attributes.WorkerType)
 	}
+
+	req4, err := http.NewRequest(http.MethodGet, ts.URL+"/jobs/triggers?worker=print", nil)
```

Comment:
Well, travis is still green. Can you add a test to check that if you ask for another `Worker`, you have 0 results?